### ISSUE TITLE
replace tags not supported

### ISSUE THREAD

**Issue by gcotone** [codeout/junoser#29]
Configuration validation fails when it contains `replace:` tags used in [load replace](https://www.juniper.net/documentation/us/en/software/junos/cli/topics/topic-map/junos-config-files-loading.html) operations.

E.g.:

```bash
# test.conf
groups {
    INTERFACE_SETTINGS {
        replace: interfaces {
            "<[xg]e*>" {
                hold-time up 5000 down 0;
            }
        }
    }
}
```

```bash
shell> junoser -c test.conf                  
Invalid syntax:  set groups INTERFACE_SETTINGS replace: interfaces "<[xg]e*>" hold-time up 5000 down 0

```

**Comment by codeout** [comment]
Thanks for your report. I haven't cared about `replace:` tag, but let me take a look.
We just need to remove them before processing probably.

**Comment by gcotone** [comment]
thanks @codeout 
Yes that should work. I'm doing a `sed -i 's/replace: //g' config.file` before running junoser and it works.

**Comment by codeout** [comment]
@gcotone Just published `v0.4.5` with the fix. Hoping it works for you.

```
$ junoser -c <<EOS
groups {
    INTERFACE_SETTINGS {
        replace: interfaces {
            "<[xg]e*>" {
                hold-time up 5000 down 0;
            }
        }
    }
}
EOS
```

If you see any further problems, please re-open this issue.
Thanks again for your bug report.
